### DISCUSSION TITLE
How to create an http connection using host and apikey?

### DISCUSSION THREAD

**Discussion by jdowthwaite** [zapadi/redmine-net-api#317] [Q&A]
Our server is not using https.  Is it possible to create a new RedmineManager object using a host and apikey?

The scheme is a readonly variable so I can't set it after creating a new RedmineManager object.

What am I missing? (yes, I know we're missing a security cert.).

**Comment by zapadi** [comment]
Hi,
You can't right now. A fix will be available soon.

**Reply by jdowthwaite** [reply]
Not sure if I'm missing something or if I've done something wrong but I'm getting a 502 error ("The remote server returned an error: (502) Bad Gateway.") with the latest version (version 4.6.0).

Using this to create my manager: 
  Dim redMgr = New RedmineManager(host, apiKey, MimeFormat.Xml, False, Nothing, Nothing, "http")

After instantiating the manager, I can get the project, status, and tracker correctly but as soon as I call  redMgr.CreateObject(newIssue), I get the 502 error.

I should note that an older build running the redmine-net452-api.dll can still submit tickets without issue.

Any help is appreciated.

**Comment by AbeYangCL** [comment]
I also have the same program , how can I set no need TLS os SSL when using RedmineManager
